frontend/src/component/feature/FeatureToggleList/FeatureToggleListActions/FeatureToggleListActions.tsx

```diff
@@ -0,0 +1,134 @@
+import { FC, useState } from 'react';
+import {
+    IconButton,
+    ListItemIcon,
+    ListItemText,
+    MenuItem,
+    MenuList,
+    Popover,
+    styled,
+    Tooltip,
+    Typography,
+} from '@mui/material';
+import { Add, FileDownload, MoreVert } from '@mui/icons-material';
+import { Link } from 'react-router-dom';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { useUiFlag } from 'hooks/useUiFlag';
+import { CREATE_FEATURE } from 'component/providers/AccessProvider/permissions';
+import { PermissionHOC } from 'component/common/PermissionHOC/PermissionHOC';
+import { useCreateFeaturePath } from 'component/feature/CreateFeatureButton/useCreateFeaturePath';
+
+const StyledActions = styled('div')(({ theme }) => ({
+    display: 'flex',
+    justifyContent: 'center',
+}));
+
+const StyledPopover = styled(Popover)(({ theme }) => ({
+    borderRadius: theme.shape.borderRadiusLarge,
+    padding: theme.spacing(1, 1.5),
+}));
+
+interface IFeatureToggleListActions {
+    setShowExportDialog: (show: boolean) => void;
```

Comment:
do we need to expose low level hook method if we only call this function with true? Maybe from the FeatureToggleListAction perspective it can be called openDialog: () => void?

Reply:
Good idea, done.